TMessagesProj/jni/tgnet/ConnectionSocket.cpp

```
@@ -146,6 +146,22 @@ static void generate_public_key(unsigned char *key) {
     BN_clear_free(mod);
 }
 
+[[nodiscard]] int PaddingExtensionLength(uint32_t currentSize) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)